src/views/edition/components/artefact-card.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <div class="card">
+        <!-- <router-link :to="{ path: `/editions/${editionId}/imaged-objects/${imagedObject.id}` }">
+            <img class="card-img-top" v-lazy="imageUrl" v-if="imageUrl" alt="Imaged-Object">
+        </router-link> -->
+        <artefact-image class="card-img-top" v-if="artefact" :artefact="artefact" :scale="0.05"></artefact-image>
+        <label>{{artefact.name}} - {{artefact.side}}</label>
+    </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { Artefact } from '@/models/artefact';
+import ArtefactImage from '../../artefact-editor/artefact-image.vue';
```

Comment:
I wasn't sure whether to use them or not. Using them lets me move entire subdirectories of components from one directory to another without changing their internal imports. On the other hand, on Windows vetur complains about differences in case when importing files, something like `c:\users\itay\...` is different from `C:\Users\Itay\...` . Obviously it's a bug (I never specify the directories outside the project), but it's fixed by moving to absolute imports.

So I guess I'll just change all imports.